### DISCUSSION TITLE
Gerenciadores de senha

### DISCUSSION THREAD

**Discussion by felipefialho** [frontendbr/forum#1317] [Discussão]
Fala pessoal, 

Faz anos que uso o **1password**, mas reparei que nos últimos anos surgiram diversas opções, inclusive gratuitas.

Gostaria de uma com integração entre todos os dispositivos.

O que vocês tem usado como gerenciador de senha?

**Comment by caioincau** [comment]
Lastpass, tenho sido bem feliz, desde que ficou de graça o sync entre devices, inclusive integra com o iOS a partir do 12.

**Comment by iamgbayer** [comment]
Também estou utilizando Lastpass a um bom tempo e sem problemas, com extensão do Chrome e app no iOS.

**Comment by gustavofsantos** [comment]
Tenho uma assinatura do LastPass e olha, eu sinto falta de nada nele, provavelmente não vou mudar de gerenciador de senhas tão cedo

**Comment by felipefialho** [comment]
To testando o LastPass, depois digo o que achei 😄

**Comment by ricalamino** [comment]
Uso LastPass pessoal e a empresa fez uma conta empresarial. Super fácil
mesclar as duas e, caso saia da empresa, tudo fica separado.

Em seg, 17 de dez de 2018 às 10:34, Felipe Fialho <notifications@github.com>
escreveu:

> To testando o LastPass, depois digo o que achei 😄
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/frontendbr/forum/issues/1317#issuecomment-447830207>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AACO2gAFnmghnTfAGjyu7g0FH-dbGtm2ks5u5485gaJpZM4ZT6hX>
> .
>


-- 

*Ricardo C. Alamino*
*Front-end Developer*
+55 11 98265 7606

**Comment by kazzkiq** [comment]
Recentemente comecei a usar o [KeeWeb](https://github.com/keeweb/keeweb) e recomendo muito, é **open-source** e aparentemente usa Electron pra rodar em todos os sistemas operacionais.

![image](https://user-images.githubusercontent.com/1953194/50111331-b2e56600-0223-11e9-8cd8-3bfdf204e435.png)

Também pode ser usado diretamente pela web, o que torna ele bastante versátil pra carregar suas senhas até em computadores de terceiros (desde que você tenha sua senha e chave privada com você, é claro).

**Comment by vNNi** [comment]
Hoje uso o [Enpass](https://www.enpass.io/), na minha opinião é bom, supri tudo que preciso e, você consegue ter um sync com um drive, por exemplo e, compartilhar as senhas em diferentes máquinas.

**Comment by felipefialho** [comment]
Bem legal heim @kazzkiq? Belissima interface.

**Comment by klarkc** [comment]
Uso o Enpass, principalmente por ser crossplataform, rodar no Chrome e Firefox e ter um custo muito baixo (apenas o app mobile é pago uma única vez).

**Comment by tcelestino** [comment]
Usei por muito tempo o Last.pass e acabei descobrindo o [bitwarden](https://bitwarden.com/) fuçando por projetos open source aqui no Github. Fiz a troca e estou bem satisfeito.
